pom.xml

```diff
@@ -281,6 +287,20 @@
                 <version>${powsyblcore.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>com.powsybl</groupId>
+                <artifactId>powsybl-security-default</artifactId>
+                <version>${powsyblcore.version}</version>
+                <type>test-jar</type>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>com.powsybl</groupId>
+                <artifactId>powsybl-security-analysis-default</artifactId>
```

Comment:
I think we should be able to not use that jar for our test:
the AFS part of the security analysis is not dependent on one implementation.